tests/pic_csv_test_class.py

```diff
@@ -3,8 +3,8 @@
 from picturae_csv_create import CsvCreatePicturae
 import picturae_config
 
-class TestCsvCreatePicturae(CsvCreatePicturae):
+class AltCsvCreatePicturae(CsvCreatePicturae):
```

Comment:
changed name to Alt, as pytest was trying to initialize initialize it as a test class